packages/website/blog/releases/announcing-v2.mdx

```diff
@@ -0,0 +1,164 @@
+
+- [Dynamic Page](https://sap.github.io/ui5-webcomponents/nightly/components/fiori/DynamicPage/): A layout component, representing a web page, consisting of a title, a header with dynamic behavior, a content area, and an optional floating footer. Use the Dynamic Page if you need to have a title, that is always visible and a header, that has configurable Expanding/Snapping functionality. 
+- [Form](https://sap.github.io/ui5-webcomponents/nightly/components/Form/): The ui5-form is a layout component that arranges labels and form fields (like input fields) pairs into a specific number of columns. And, there is also "grouping" available to assist the implementation of richer UIs. The Form component reacts and changes its layout on predefined breakpoints.
+- [Table](https://sap.github.io/ui5-webcomponents/nightly/components/Table/): The ui5-table component provides a set of sophisticated features for displaying and dealing with vast amounts of data in a responsive manner. This Table replaces the previous Table web component, that has been part of @ui5/webcomponents version 1.x. For compatibility reasons, we moved the previous Table implementation to the @ui5/webcomponents-compat package and will be maintained until the new Table is experimental.
+- [Text](https://sap.github.io/ui5-webcomponents/nightly/components/Text/): The ui5-text component displays text that can be used in any content area of an application. By default, the text will wrap when the space is not enough. IIn addition, the component supports truncation via the max-lines property, which defines the number of lines the text should wrap before it starts truncating.
+- [Tokenizer](https://sap.github.io/ui5-webcomponents/nightly/components/Tokenizer/): A ui5-tokenizer is an invisible container for ui5-tokens that supports keyboard navigation and token selection.
+- [AI Button](https://sap.github.io/ui5-webcomponents/nightly/components/ai/Button/): The ui5-ai-button component represents a button used in AI-related scenarios. It enables users to trigger actions by clicking or tapping the ui5-ai-button, or by pressing certain keyboard keys, such as Enter.
+- [AI Prompt Input](https://sap.github.io/ui5-webcomponents/nightly/components/ai/PromptInput/): The ui5-ai-prompt-input component allows the user to write custom instructions in natural language so that AI is guided to generate content tailored to user needs.
+- [Custom Option](https://sap.github.io/ui5-webcomponents/nightly/components/OptionCustom/): The ui5-option-custom component defines a custom content of an option in the ui5-select. The component accepts arbitrary HTML content to allow full customization.
+- [Custom Suggestion Item](https://sap.github.io/ui5-webcomponents/nightly/components/SuggestionItemCustom/): The ui5-suggestion-item-custom is a type of suggestion item, that can be used to place suggestion items with custom content in the input. 
+
```

Comment:
Perhaps the `ListItemGroup` could also be added here. The `new` tag currently is missing and a PR is in review https://github.com/SAP/ui5-webcomponents/pull/9380